src/components/shared/custom-inputs/styles.js

```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
+
+export const Input = styled.input`
+  border: 1.5px solid ${(props) => (props.invalid ? "red" : "#c4c4c4")};
```

Comment:
dobra, po co to jest? :D jak invalid w reactstrapie ma defaultowo invalid na czerwoną obramówkę + czerwony X